src/site/content/en/learn/css/index.md

```diff
@@ -1,313 +1,37 @@
 ---
-title: Overview
+title: Learn CSS
 authors:
-  - una
-description: This post highlights a few powerful lines of CSS that do some serious heavy lifting and help you build robust modern layouts.
-date: 2020-07-07
-hero: image/admin/B07IzuMeRRGRLH9UQkwd.png
-alt: Holy grail layout.
-audio:
-  title: 'The CSS Podcast - 001: Flexbox'
-  src: 'https://traffic.libsyn.com/secure/thecsspodcast/TCP035_v2.mp3?dest-id=1891556'
-  thumbnail: image/foR0vJZKULb5AGJExlazy1xYDgI2/ECDb0qa4TB7yUsHwBic8.png
+  - andybell
+description: A course that breaks down the fundamentals of CSS to help you become a well-rounded front-end developer.
```

Comment:
```suggestion
description: An evergreen CSS course and reference to level-up your web styling expertise.
```

Reply:
What do you all think about this for the description? We use the works "breaks down fundamentals" and "well-rounded front-end developer" in the intro text, so i updated the description text with the course pitch.